samples/app-sso/nodejs/README.md

```diff
@@ -0,0 +1,184 @@
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+
+3. To test locally, you'll need [Ngrok](https://ngrok.com/) installed on your development machine.
+Make sure you've downloaded and installed Ngrok on your local machine. ngrok will tunnel requests from the Internet to your local computer and terminate the SSL connection from Teams.
+
+> NOTE: The free ngrok plan will generate a new URL every time you run it, which requires you to update your Azure AD registration, the Teams app manifest, and the project configuration. A paid account with a permanent ngrok URL is recommended.
+
+4. Required Permissions
+        * Make sure you have the following Graph permissions enabled: `email`, `offline_access`, `openid`, `profile`, and `User.Read` (default).
+        * For permissions `Manage > API Permissions`
+        * Our SSO flow will give you access to the first 4 permissions, and we will have to exchange the token server-side to get an elevated token for the `profile` permission (for example, if we want access to the user's profile photo).
+        
+![image](https://user-images.githubusercontent.com/85108465/121638666-f80f4980-caa8-11eb-9b75-09b0e86c6d6a.png)
+
+## To try this sample
+
+- Setup for Bot SSO
+Refer to [Bot SSO Setup document](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation-sso-quickstart/BotSSOSetup.md).
+
+> **NOTE:**
+Create [Bot Framework registration resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration) in Azure
+    - Use the current `https` URL you were given by running ngrok. Append with the path `/api/messages` used by this sample
+    - Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+    - __*If you don't have an Azure account*__ you can use this [Bot Framework registration](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/create-a-bot-for-teams#register-your-web-service-with-the-bot-framework)
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- In a console, navigate to `samples/app-sso/nodejs`
+
+    ```bash
+    cd samples/app-sso/nodejs
+    ```
+
+- Run ngrok - point to port `4000`
+
+    ```bash
+    ngrok http -host-header=localhost 4000
+    ```
+
+
+- Update the `.env` configuration for the bot to use the `MicrosoftAppId` (Microsoft App Id), `MicrosoftAppPassword` (App Password) and `connectionName` (OAuth Connection Name) from the Bot Framework registration. 
+> NOTE: the App Password is referred to as the `client secret` in the azure portal and you can always create a new client secret anytime.
+
+
+- Install modules & Run the Server `Bot, ME`
+Open a terminal and navigate to project root directory
```

Comment:
Is it possible to combine these under same project for simplicity?

Reply:
PR has been updated with Updated Readme and Screen shots, please have a look

Reply:
Updated PORT to 3978 and readme file